source/common/http/filter_manager.cc

```diff
@@ -824,7 +836,13 @@ void FilterManager::sendLocalReply(
 
   stream_info_.setResponseCodeDetails(details);
 
-  filter_manager_callbacks_.onLocalReply(code);
+  StreamFilterBase::LocalReplyData data{code, details};
+  if (FilterManager::onLocalReply(data) == LocalErrorStatus::ContinueAndResetStream) {
+    ENVOY_STREAM_LOG(debug, "Resetting stream due to {}. onLocalReply requested reset.", *this,
+                     details);
```

Comment:
Any way we can capture which filter is triggering the reset? Or let the filter that triggers a reset include information?

Reply:
At least with local replies we can infer it from the response code details, but yea I don't think this is necessary, just a nice to have